config/default.json

```diff
@@ -182,7 +182,7 @@
     //    deny: users cannot track for everything and must track against individual pokemon
```

Comment:
does default.json no longer work because of comments in json file?

Reply:
No all fine. Node config supports json comments